### ISSUE TITLE
您好，mkv内嵌字幕不支持，代码中有不少被#ifdef FFP_MERGE屏蔽了，请问这个宏可以通过编译打开么？

### ISSUE THREAD

**Issue by yangchengtest** [Bilibili/ijkplayer#2224]
这个宏请问怎么打开编译？
还是说设计中就是用这个宏屏蔽不成熟代码的呢？
多谢！～

**Comment by bbcallen** [comment]
No, I don't need it, so I hide it. That's to say they are never tested in ijkplayer.
The macros are left there for the convenience of merging from official ffplay.

**Comment by zhaodp** [comment]
最好能支持内嵌字幕，vitamio对内嵌字幕支持的非常好
